101-app-service-docs-windows/azuredeploy.json

```diff
@@ -0,0 +1,125 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "webAppName": {
+            "type": "string",
+            "defaultValue": "[concat('webApp-', uniqueString(resourceGroup().id))]",
+            "metadata": {
+                "description": "Web app name."
+            },
+            "minLength": 2
+        },
+        "location": {
+            "type": "string",
+            "defaultValue": "[resourceGroup().location]",
+            "metadata": {
+                "description": "Location for all resources."
+            }
+        },
+        "sku": {
+            "type": "string",
+            "defaultValue": "F1",
+            "metadata": {
+                "description": "The SKU of App Service Plan."
+            }
+        },
+        "language": {
+            "type": "string",
+            "defaultValue": ".net",
+            "allowedValues": [
+                ".net",
+                "php",
+                "node",
+                "html"
+            ],
+            "metadata": {
+                "description": "The language stack of the app."
+            }
+        },
+        "helloWorld": {
+            "type": "bool",
+            "defaultValue": "[bool(0)]",
```

Comment:
since this is a bool, use true/false (the expression adds unnecessary complexity)...  could also use "yes/no" but then you need to use allowed values

Reply:
sounds good